awx/main/models/jobs.py

```diff
@@ -759,7 +755,7 @@ def display_artifacts(self):
 
     @property
     def can_run_containerized(self):
-        return any([ig for ig in self.preferred_instance_groups if ig.is_container_group])
+        return True
```

Comment:
Calling out a notable change here. I agree with it, but will make sure to follow the use of the property to double-check.

Reply:
Finished looking, I feel pretty confident in this.

Reply:
Grep shows 5 instances:

```
./awx/main/scheduler/task_manager.py:282:                elif task.can_run_cont>
./awx/main/scheduler/task_manager.py:485:                if task.can_run_contai>
./awx/main/models/jobs.py:757:    def can_run_containerized(self):
./awx/main/models/ad_hoc_commands.py:154:    def can_run_containerized(self):
./awx/main/models/unified_jobs.py:741:    def can_run_containerized(self):
```

3 are the defs, 1 is the one I know I moved this `any()` check to.  The other one (`./awx/main/scheduler/task_manager.py:485`) seems to be doing this same `.is_container_group` logic in a more complex loop.